### ISSUE TITLE
Specifying the account names when authenticating via LDAP

### ISSUE THREAD

**Issue by timothyjmtan** [Erudika/scoold#133]
Hi,

Currently I have enabled LDAP authentication. 
The users are logging in with their email address, e.g. **user1**@company.com.

When they successfully login, I noticed that the accounts that are automatically created for them are not **user1**. I would like it to be as so.

Which property in `application.conf` should I specify for this? Thanks.

**Comment by albogdano** [comment]
I don't quite understand what you need exactly. Do you mean that names are different than their email nicknames? The name of a person is pulled from LDAP and has nothing to do with their email address.

**Comment by timothyjmtan** [comment]
Hi Alex,

Yup. So what we are seeing now is that even though they are logging in with
their email address such as user1@company.com, the usernames seen in scoold
are their CN attributes such as 'DRADUSN' . We would like the usernames to
instead be 'user1' , which is the sAMAccountName attribute.

Hope that makes sense.

**Comment by albogdano** [comment]
Ok, that makes sense. I'll see what I can do.

**Comment by timothyjmtan** [comment]
Thanks!

In the meantime I did a quick fix on my end. I changed this line in the `para-server` module.

**LdapAuthFilter.java** (line 115)
`String name = email.split("@")[0];`

As Spring has limited getter methods, I am just splitting the username out from the email address and using that instead. Ideally Spring would have provided a `.getMailNickname()` or a `.getsAMAccountName()` to access the values directly.
